HEADER.js

```diff
@@ -118,15 +118,6 @@ fabric.charWidthsCache = { };
  */
 fabric.textureSize = 2048;
 
-/**
- * When 'true', style information is not retained when copy/pasting text, making
- * pasted text use destination style.
- * Defaults to 'false'.
- * @type Boolean
- * @default
- */
-fabric.disableStyleCopyPaste = false;
```

Comment:
i think we need this and i think is not hard to keep it in place. Flexible or not. Can be moved to canvas level if it helps

Reply:
I don't care about this. It is dead code AFAIC.

Reply:
It is not used anywhere after this PR